http.go

```diff
@@ -17,23 +17,37 @@ import (
 type httpServer struct {
 	ctx *Context
 
-	Requests     uint64
-	GetRequests  uint64
-	GetHits      uint64
-	GetMisses    uint64
+	Requests uint64
```

Comment:
I think so. It's fine though -- we've basically doubled the amount of fields in this struct so we should break it up more.